components/ChangeSigner/index.tsx

```diff
@@ -40,13 +40,10 @@ export const ChangeSigner = ({
   const [txState, setTxState] = useState<TxState>(TxState.FillingForm)
   const [txFee, setTxFee] = useState<FilecoinNumber | null>(null)
 
-  // Get signer addresses without current wallet owner
+  // Get signer addresses, including current wallet owner
```

Comment:
@navFooh do you see any issue with this? you should be able to propose to swap yourself out

Reply:
Looks good to me! Should we add a warning in this case, saying that you're about to remove yourself from the safe?